hotsos/core/ycheck/engine/properties/requires/types/apt.py

```diff
@@ -28,32 +28,38 @@ def installed_versions(self):
 
         return _versions
 
-    def package_version_within_ranges(self, pkg, versions):
-        result = False
-        versions = sorted(versions, key=lambda i: str(i['min']), reverse=True)
+    def package_version_within_ranges(self, pkg, version_criterias):
```

Comment:
nit:

singular: criterion
plural: criteria

Reply:
I always assumed that the word 'criteria' was singular :) Thanks for the correction!

Reply:
Done